test/chainsaw/integration/TTLSecondsAfterFinished-in-jobs/chainsaw-test.yaml

```diff
@@ -11,3 +11,14 @@ spec:
             file: 00-install.yaml
         - assert:
             file: 00-assert.yaml
+      catch:
```

Comment:
Are you sure that keptnworkloadinstance and keptnapp are needed here? AFAIK they are not created in this test (please check 00-install.yaml). Also metrics-operator is not used here